rero_ils/modules/notifications/tasks.py

```diff
@@ -126,6 +126,8 @@ def create_notifications(types=None, tstamp=None, verbose=True):
                     )
         process_notifications(NotificationType.OVERDUE)
     notification_sum = sum(notification_counter.values())
+    # in timestamp we need also 0 values
+    set_timestamp('notification-creation', **counters)
     counters = {k: v for k, v in notification_counter.items() if v > 0}
```

Comment:
You tried to use `**counters` before it is initialized.
Both line should be switched